it/mail/chrome/messenger/filter.properties

```diff
@@ -1,71 +1,71 @@
-invalidCustomHeader=Uno dei filtri utilizza un'intestazione personalizzata contenente un carattere non valido, come ad esempio ':', un carattere non stampabile, un carattere non ASCII o un carattere ASCII a 8 bit. Aprire il file che contiene i filtri (msgFilterRules.dat) con un editor di testi e rimuovere i caratteri non validi dalle intestazioni personalizzate.
-continueFilterExecution=L'applicazione del filtro %S non è riuscita. Continuare ad applicare i filtri?
-promptTitle=Applicazione filtri messaggi
-promptMsg=Applicazione dei filtri in corso.\nContinuare ad applicare i filtri?
+mustSelectFolder=Devi selezionare una cartella di destinazione.
+enterValidEmailAddress=Inserisci un indirizzo e-mail valido a cui inoltrare.
+pickTemplateToReplyWith=Scegli un modello con cui rispondere.
+mustEnterName=Devi assegnare un nome a questo filtro.
+mustSelectAction=Devi selezionare almeno un'azione filtro.
+cannotHaveDuplicateFilterTitle=Nome filtro duplicato
+cannotHaveDuplicateFilterMessage=Il nome filtro inserito seiste già. Inserisci un nome filtro diverso.
+deleteFilterConfirmation=Sei sicuro di voler eliminare il filtro o i filtri selezionati?
+untitledFilterName=Filtro senza titolo
+filterListBackUpMsg=I tuoi filtri non funzionano perché il file msgFilterRules.dat, che contiene i tuoi filtri, non è leggibile. Sarà creato un nuovo file msgFilterRules.dat; e sarà creata una copia del file originale, nominata rulesbackup.dat, nella stessa cartella.
+customHeaderOverflow=Hai superato il limite di 50 intestazioni personalizzate. Rimuovi una o più intestazioni personalizzate e riprova.
+filterCustomHeaderOverflow=I tuoi filtri hanno superato il limite di 50 intestazioni personalizzate. Modifica il file msgFilterRules.dat, che contiene i tuoi filtri, per utilizzare meno intestazioni personalizzate.
+invalidCustomHeader=Uno dei tuoi filtri utilizza un'intestazione personalizzata che contiene un carattere non valido, come ':', un carattere non stampabile, un carattere non ASCII o un carattere ASCII a 8 bit. Modifica il file msgFilterRules.dat, che contiene i tuoi filtri, per rimuovere i caratteri non validi dalle intestazioni personalizzate.
+continueFilterExecution=L'applicazione del filtro %S non è riuscita. Vuoi continuare ad applicare i filtri?
+promptTitle=Esecuzione filtri
+promptMsg=Stai attualmente filtrando i messaggi.\nVuoi continuare ad applicare i filtri?
 stopButtonLabel=Interrompi
 continueButtonLabel=Continua
-cannotEnableFilter=Questo filtro è stato probabilmente creato da una versione successiva di Mozilla/Netscape. Non è possibile attivare questo filtro in quanto il formato non è supportato.
+cannotEnableFilter=Il filtro è stato probabilmente creato da una versione successiva di mozilla/netscape. Non puoi attivare questo filtro in quanto il formato non è supportato.
```

Comment:
CORRECTIONS NEEDED. 

Mozilla and Netscape are registered trademarks, and need to be capitalized.